DTS-STN/Service-Canada-Labs
# Update VA Overview Page

# Description

[Update Virtual Assistant overview page](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=71476)

Update Virtual assistant overview page to match the new design on [Figma](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?node-id=10646%3A27673)

## Acceptance Criteria

Virtual assistant page match the Figma design

## Test Instructions

1. yarn dev
2. visit /project/virtual-assistant
3. compare the page with the design on Figma
4. check button and links (The blog page doesn't exist atm, will get 404 when you try to go to the blog post)

## Checklist
